docs/tutorial/install.rst

```diff
@@ -7,9 +7,9 @@ Setup VMs
 
 .. code-block:: shell
 
-    lxc launch images:ubuntu/22.04/cloud microceph-1 --vm
-    lxc launch images:ubuntu/22.04/cloud microceph-2 --vm
-    lxc launch images:ubuntu/22.04/cloud microceph-3 --vm
+    lxc launch images:ubuntu/22.04/cloud microceph-1 --vm --type m1.large
+    lxc launch images:ubuntu/22.04/cloud microceph-2 --vm --type m1.large
+    lxc launch images:ubuntu/22.04/cloud microceph-3 --vm --type m1.large
```

Comment:
If I understand it correctly, m1.large comes with 2 vCPU cores and 7.5GB memory. Which means that this tutorial requires 6 vCPU and 22.5GB memory in total.

From a Ceph point of view, this requirement is absolutely correct since `osd_memory_target` for example is set to 4GB (per OSD) by default. At the same time, if somebody would like to test **micro**ceph quickly, then most laptops in the market gets out of the picture.

From an user story view, it would be nice if there is an opt-in feature to fit into 16GB of memory by having a lower memory per VM like 4GB and setting osd_memory_target as 2GB or something by microceph.

Reply:
My usual setup is `-c limits.cpu=2 -c limits.memory=4GiB` which works just fine for development and demos.

While Ceph can indeed use up to 4GiB per OSD unless configured otherwise, it won't actually do that instantly, it's just a maximum that it's allowed to reach and in my experience pretty rarely reaches even in production environment.

So I suspect that using `-c limits.cpu=2 -c limits.memory=4GiB` is probably a better alternative here as that will still fit on systems that have 16GiB of RAM. Worth noting that similar to Ceph, LXD won't reserve the memory, so in practice you'll be running well below that potential 12GiB.